CHANGELOG.md

```diff
@@ -20,7 +20,6 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 - Add PhysicalMachine CRD [#2587](https://github.com/chaos-mesh/chaos-mesh/pull/2587)
 - New command `physical-machine` to `chaosctl` [#2624](https://github.com/chaos-mesh/chaos-mesh/pull/2624)
 - Add status "Deleting" for chaos experiments on Chaos Dashboard [#2708](https://github.com/chaos-mesh/chaos-mesh/pull/2708)
-- Add time skew for gettimeofday [#2742](https://github.com/chaos-mesh/chaos-mesh/pull/2742)
```

Comment:
I think the deletion is not required. Because changelog uses different sections for different branches. And these changes on the master branch are indeed not released in any version. 🤔

Reply:
Ohh, I get it. 

> And these changes on the master branch are indeed not released in any version.

maybe 'these changes on the master branch are indeed not released in next major version' is more accurate, because #2742 is already released in v2.1.4. 🤔

Reply:
Fixed!